### PR TITLE
remove duplicate and unused tags

### PR DESCRIPTION
**Pull request by seuros**

### CODE REVIEW

File: db/migrate/20150103105333_tag_aato_cleanup.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class TagAatoCleanup < ActiveRecord::Migration
+  def up
+   ActsAsTaggableOn::Tag.delete_all(taggings_count: 0)
+   ActsAsTaggableOn::Tag.destroy_all(name: '')
+   ActsAsTaggableOn::Tag.destroy_all(name: %w(navarro etagwerker mattvvhat Assembly angels  capital combinations turbulenz semerda))
```
Comment (rohitpaulk) [inline db/migrate/20150103105333_tag_aato_cleanup.rb]: Why specifically these?

Comment (seuros) [inline db/migrate/20150103105333_tag_aato_cleanup.rb]: These are the duplicates in our case in production.

Comment (rohitpaulk) [inline db/migrate/20150103105333_tag_aato_cleanup.rb]: Why are we removing all of them then? Shouldn't we delete all but one and update the corresponding `tag_id`s in the `taggings` table?

Comment (seuros) [inline db/migrate/20150103105333_tag_aato_cleanup.rb]: They are not real tags. I will have fixed them if it was jquery, ruby, php...

Comment (rohitpaulk) [inline db/migrate/20150103105333_tag_aato_cleanup.rb]: Okay, that makes sense.

**Pull request merged**